.changeset/grumpy-news-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/select': minor
+---
+
+Consuming applications can now set the width of the select dropdown by passing a value to width through the `className` prop.
```

Comment:
This was possible before. This just makes the select fill the available space.

I'm a tad concerned that this PR doesn't address PD-1006 at all...